p2p/pkg/preconfirmation/tracker/tracker.go

```diff
@@ -538,6 +544,24 @@ func (t *Tracker) openCommitments(
 				"providerAddress", commitment.ProviderAddress,
 				"winner", newL1Block.Winner,
 			)
+			if commitment.BidderAddress == nil {
```

Comment:
Same as above. Double check can be avoided.

Reply:
Also necessary since pointer types, I've made this part consistent with the other section https://github.com/primev/mev-commit/commit/394ad8913307007d197b7fba7f13010fcff047a7